### DISCUSSION TITLE
GitHub App Renovate is requesting additional access to your organization

### DISCUSSION THREAD

**Discussion by CommanderStorm** [renovatebot/renovate#23939] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I got this email at 16:46:
![image](https://github.com/renovatebot/renovate/assets/26258709/c642c644-df60-472c-987c-4f712c54c9c7)

What feature/bug is this fixing? (I have tried looking in all the places where such an answer could be found and could not find one)

### What you tried so far.

I looked 
- the previosly merged/closed PRs/issues
- relevant issues given the obvious search terms
- recent releases
- documentation
  - [faq](https://docs.renovatebot.com/faq/)
  - [Security and Permissions](https://docs.renovatebot.com/security-and-permissions/#security-and-permissions)

### Relevant debug logs

the app dashboard is not working for me (cannot log in)

**Comment by rarkins** [comment, marked as answer]
You didn't try clicking this yet? It describes the change (Package registry read rights)

![image](https://github.com/renovatebot/renovate/assets/6311784/16059947-b8e3-4d22-8e6f-a95b077b95a9)

**Reply by CommanderStorm** [reply]
Sorry for the noob question. (I never experienced this in the past ^^)
I did not expect a developer note, why this is change is done to be there.

**Reply by CommanderStorm** [reply]
Plus I misread this as "Renovate now wants the permission to accept/reject PRs"

**Reply by rarkins** [reply]
No problem, GitHub's UX is a bit "alarming"
